apps/api/src/app/users/model/user.model.ts

```diff
@@ -0,0 +1,32 @@
+import {UserRole} from "./user-role";
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class UserModel {
+
+  @PrimaryGeneratedColumn('uuid')
+  id?: string;
+
+  @Column({ nullable: false })
+  name: string;
+
+  @Column({ nullable: true })
+  email: string;
+
+  @Column({ nullable: true })
+  password: string;
+
+  @Column({ nullable: true, enum: UserRole, type: 'enum' })
+  roles: UserRole[];
```

Comment:
**issue:** set the nullable to `false` and make the default to be `User`. A user should always have a role in the application.